### ISSUE TITLE
A question about Chamfer Distance

### ISSUE THREAD

**Issue by ChenBarryHu** [yuxumin/PoinTr#55]
Hi Xumin,

This is my second "issue ticket" already. :) Thank you so much again for helping with my last issue!

If you don't mind, my questions this time are: 
1. When using Chamfer Distance as the loss function, could the dense loss go down to zero when overfitting to a single sample?
2. I tried overfitting the PoinTr model on one single sample, here is the training loss: note the red curve is trained with both dense and sparse loss, and the grey curve is trained with only dense loss. I tried with various lr actually, but couldn't get a zero dense loss, so I wonder is zero dense loss even possible?
![Screenshot from 2022-05-13 21-44-05](https://user-images.githubusercontent.com/17330047/168378975-89b803b1-f43e-4a7a-958a-1d9eadaa03e8.png)

Thank you so much and have a great weekend!

Barry

**Comment by yuxumin** [comment]
Hi, nice to see you again and sorry for the late reply.
I haven't tried to optimize the model with only one single shape. But i think the result you provide is reasonable. 
Actually, the optimization target for CD is changing during each round (because the correspendence between predicted points and gt points are built upon the distance in L2-space for each round), so it is hard to optimize it to a special case (like CD=0). 
In this case, a pre-defined correspendence and a L2 Loss can easily help the model to overfit on one single shape, in my opinion.

**Comment by ChenBarryHu** [comment]
I agree, thank you!
